src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/configuration/HBaseConfiguration.kt

```diff
@@ -7,10 +7,12 @@ import org.apache.hadoop.hbase.client.ConnectionFactory
 import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
 import uk.gov.dwp.dataworks.logging.DataworksLogger
 
 @Configuration
 @ConfigurationProperties(prefix = "hbase")
+@Profile("HBASE")
```

Comment:
Is there an associated infra change in place to activate this?

Reply:
Yeh I have that on the side, will post that shortly before this gets merged

Reply:
Cool